src/components/app/routes/data/utils.js

```diff
@@ -45,13 +43,11 @@ export async function ensureEnterpriseAppData({
   const matchedBFFQuery = resolveBFFQuery(requestUrl.pathname);
   const enterpriseAppDataQueries = [];
   if (!matchedBFFQuery) {
-    const subscriptionsQuery = querySubscriptions(enterpriseCustomer.uuid);
     enterpriseAppDataQueries.push(
       // Enterprise Customer User Subsidies
-      safeEnsureQueryData({
+      safeEnsureQueryDataSubscriptions({
         queryClient,
-        query: subscriptionsQuery,
-        fallbackData: getBaseSubscriptionsData().baseSubscriptionsData,
+        enterpriseCustomer,
```

Comment:
[inform] Migrates to using the query-specific helper functions wrapping `safeEnsureQueryData` so this code path doesn't need to worry about which `fallbackData` should be provided, etc.